Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Isotropic_remeshing/remesh_impl.h

```diff
@@ -859,6 +875,12 @@ namespace internal {
         vvb -= 1;
         vvc += 1;
         vvd += 1;
+#ifdef VALE
```

Comment:
Do you want to keep that ? you observe no gain when activating it?

Reply:
In fact THAT is the code we must take.